pan-domain-auth-verification/src/main/scala/com/gu/pandomainauth/package.scala

```diff
@@ -0,0 +1,7 @@
+package com.gu
+
+package object pandomainauth {
+  case class PublicKey(key: String)
```

Comment:
If you're just using a type as a wrapper like this have it `extend AnyVal` so it gets unpacked at runtime.